config/foreman.migrations/20210625142707_dynamic_puppet-in-foreman-groups.rb

```diff
@@ -0,0 +1,5 @@
+if answers['foreman'].is_a?(Hash)
```

Comment:
The file naming convention is to use underscores (except where we messed up)

Reply:
This was what the script generated for me but I can change it.

Reply:
Please do change it.